### DISCUSSION TITLE
Freeze columns and edit columns on the fly

### DISCUSSION THREAD

**Discussion by xzhenghao** [rappasoft/laravel-livewire-tables#1612] [Q&A]
Hi,

Any idea if we could do the above using this package? Thank you.

**Comment by lrljoe** [comment]
Not possible at present to freeze Columns nor Edit on-the-fly.

Freezing Columns is on the roadmap.
Editing on-the-fly isn't on my roadmap at the moment (may be on @rappasoft directly), but I'd always suggest that you use other methods that are cleaner for the end-user (for example Modals).
